Makefile

```diff
@@ -17,3 +20,16 @@ tidy:
 .PHONY: proto
 proto: tools
 	buf generate
+
+.PHONY: build
+build:
+	mkdir -p ./bin
+	go build -o bin/pomerium-cli cmd/*.go
+
+.PHONY: clean
+clean:
+	rm -f bin/*
+
+.PHONY: snapshot
+snapshot:
+	APPARITOR_GITHUB_TOKEN=foo goreleaser release --snapshot --rm-dist
```

Comment:
```suggestion
	APPARITOR_GITHUB_TOKEN=foo goreleaser release --snapshot --rm-dist
```

foo? :)

Reply:
So it doesn't yell at you for not having a token set.